testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -1488,6 +1489,21 @@ QUnit.module('Options', {
         assert.strictEqual(cellContent, 'Custom template', 'cell content is correct after cellTemplate runtime change');
     });
 
+    QUnit.test('cellTemplate is rendered fow week cell', function(assert) {
+        this.calendar.option({
```

Comment:
I want to know which cellData is for 'week' cells? Can we check it?

Reply:
Yes, week number cells have `cellIndex: -1` in screenshot test it's shown how to make week cells bold and others cell italic